### PR TITLE
update jersey to version 2

### PR DESCRIPTION
**Pull request by chrisdoc**
add gitignore
update jersey to version 2
updating WNSClient to use jersey 2 and jackson 2
add junit 4 tests
add gradle support
cleanup

### CODE REVIEW

**Comment by fernandospr** [comment]
Please, modify the pom from 1.4-SNAPSHOT to 1.3.1-SNAPSHOT as this would not be a release with new features, these are internal changes.

**Comment by chrisdoc** [comment]
Hey @fernandospr thank you for your comments. I removed the test and change the instance of to getType(). I also changed the version number however I did not change it to 1.4-SNAPSHOT in the first place. This is already been set in the tip of master

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	<groupId>com.github.fernandospr</groupId>
 	<artifactId>java-wns</artifactId>
 	<packaging>jar</packaging>
-	<version>1.4-SNAPSHOT</version>
+	<version>1.3.1</version>
```
Comment (fernandospr) [inline pom.xml]: This must be 1.3.1-SNAPSHOT because this is still under development.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,31 @@
+apply plugin: 'java'
+
+group = 'com.github.fernandospr'
+version = '1.3.1'
```
Comment (fernandospr) [inline build.gradle]: This must be 1.3.1-SNAPSHOT because this is still under development.

**Comment by chrisdoc** [comment]
PING

**Comment by fernandospr** [comment]
Sorry @chrisdoc for the delay, I have decided not to use Gradle. 
I don't plan to mantain the maven and gradle files.
For now, I would like to stick to maven.
If you want you can fork this repository and use gradle only.

**Comment by chrisdoc** [comment]
@fernandospr no problem, I removed the gradle stuff. I just feel more comfy with Gradle then Maven :smile:

**Comment by fernandospr** [comment]
@chrisdoc People are getting problems when I merged your PR... It seems that it is not serializing the list of objects the way it used to be. Please check issue: [12](https://github.com/fernandospr/java-wns/issues/12)
Please can you help me solve the issue?

**Pull request merged**